src/universal/conf/network.conf

```diff
@@ -71,21 +71,21 @@ mantis {
 
       # Path to the keystore storing the certificates (used only for https)
       # null value indicates HTTPS is not being used
-      certificate-keystore-path = "conf/mantis.jks"
+      # certificate-keystore-path = null
 
       # Type of certificate keystore being used
       # Has to be null or one of the ones listed in:
       #   https://docs.oracle.com/javase/8/docs/technotes/guides/security/StandardNames.html#KeyStore
       # A null value indicates HTTPS is not being used
-      certificate-keystore-type = "jks"
+      # certificate-keystore-type = null
 
       # File with the password used for accessing the certificate keystore (used only for https)
       # null value indicates HTTPS is not being used
-      certificate-password-file = "conf/nopassword"
+      # certificate-password-file = null
 
       # Enabled JSON-RPC APIs over the JSON-RPC endpoint
       # Available choices are: eth, web3, net, personal, daedalus
-      # apis = "eth,web3,net,personal,daedalus"
+      apis = "eth,web3,net,personal"

```

Comment:
no `#`?

Reply:
i disabled daedalus for cli....

Reply:
@LukasGasior1 @KonradStaniec Any foresee an issue with disabling the daedalus API for the command line ...?

Reply:
nope